### ISSUE TITLE
validation loss 

### ISSUE THREAD

**Issue by Tortoise17** [leimao/Singing_Voice_Separation_RNN#2]
Dear friend. 

I have one more question. 
no matter how much the model is trained. the loss function does not go below 1.2 or we can say 1.0. 

Can you guide why? or this is multiplied by any of the factor?

Because as far as I understand, for good training, the loss should decrease 0.1 or at least nearby. 

If you can guide because I am not so expert and trying to learn. 

Greetings.

**Comment by leimao** [comment]
You are right. We want to get low validation loss. However, probably because this task is hard and our model is not perfect, 1.0 is the lowest validation loss it can achieve given the configurations. Note that talking the absolute loss without unit or its physical context does not make sense. It is incorrect to say for good training, the loss should decrease to 0.1 or lower.

**Comment by Tortoise17** [comment]
You are right. absolutely. That what I am trying to understand here the unit for loss.  or physical context. !! Thank you !!
